llama_stack/cli/stack/run.py

```diff
@@ -59,6 +66,11 @@ def _add_arguments(self):
             help="Image Type used during the build. This can be either conda or container or venv.",
             choices=[e.value for e in ImageType],
         )
+        self.parser.add_argument(
+            "--no-ui",
+            action="store_true",
+            help="Don't start the UI server",
+        )
```

Comment:
Can we have no UI by default?

Reply:
@terrytangyuan can you share your concern?

Reply:
This would introduce new UI dependencies that might require more maintenance efforts, e.g. security patches.